README.md

```diff
@@ -1,7 +1,30 @@
-# Toggle quotes package [![Build Status](https://travis-ci.org/atom/toggle-quotes.svg?branch=master)](https://travis-ci.org/atom/toggle-quotes)
+# Toggle Quotes
 
-`ctrl-"` (or `cmd-"` on OS X) to toggle a single-quoted string to a double-quoted string, and vice
+[![Build Status](https://travis-ci.org/atom/toggle-quotes.svg?branch=master)](https://travis-ci.org/atom/toggle-quotes)
+[![Dependency Status](https://david-dm.org/atom/toggle-quotes.svg)](https://david-dm.org/atom/toggle-quotes)
+
+Toggle a single-quoted string to a double-quoted string, and vice
 versa. Available when using any grammar that supports single-quoted and
-double-quoted strings (e.g., JavaScript, Python, Ruby, etc.).
+double-quoted strings (e.g. JavaScript, Python, Ruby, etc.).
 
 ![toggle-quotes-demo](https://f.cloud.github.com/assets/2988/1764634/c1098d1e-6729-11e3-88f4-73cc336c0173.gif)
+
+This is a package for [Atom](https://atom.io), a hackable text editor for the 21st Century.
+
+## Usage
+
+Search for `Toggle Quotes: Toggle` in the Command Palette or enter the keybinding for your system to toggle quotes on a string. Only available when using any grammar that supports single-quoted and double-quoted strings (e.g. JavaScript, Python, Ruby, etc.).
```

Comment:
Maybe add a few words to say that the cursor needs to be positioned between the quote characters before executing the command? Perhaps something like:

> Put the cursor within a string and execute the `Toggle Quotes: Toggle` command to toggle between single and double quotes.

I also :scissors: the part about using the command palette and keybindings because we have sections on those below, and it's not really important how the command is executed maybe, just that it _is_?